webpack.prod.config.babel.js

```diff
@@ -27,7 +24,9 @@ for (const app of appsBuildList) {
 const settings = {
   devtool: 'source-map',
   context: path.resolve(__dirname),
-  progress: true,
+  devServer: {
+    progress: true,
```

Comment:
Yeah, it's just for the webpack dev server. I can just move it to the dev config though.